WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.java

```diff
@@ -62,7 +63,12 @@ public void onCreate(Bundle savedInstanceState) {
             return;
         }
 
-        mPostList = (PostsListFragment) fm.findFragmentById(R.id.postList);
+        mPostList = PostsListFragment.newInstance(mSite);
+
```

Comment:
It shouldn't be necessary to create the fragment when the activity is recreated. Instead you can find the existing fragment, and if it's not there create it. Ex:

```
mPostList = (PostsListFragment) getFragmentManager().findFragmentByTag(PostsListFragment.TAG);
        if (mPostList == null) {
            mPostList = PostsListFragment.newInstance(mSite);
            getFragmentManager().beginTransaction()
                    .add(R.id.post_list_container, mPostList, PostsListFragment.TAG)
                    .commit();
        }
```